src/server_manager/web_app/cloud_accounts.ts

```diff
@@ -1,4 +1,4 @@
-// Copyright 2018 The Outline Authors
```

Comment:
Copyright should be followed by the year of first publication. I think we should keep the original years.

Reply:
Github shows it as the same file that got renamed.